hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/Counters.java

```diff
@@ -15,5 +14,11 @@
             FILTERED = "filtered." + GROUP + "." + TOPIC + "." + SUBSCRIPTION,
             INFLIGHT = "inflight." + GROUP + "." + TOPIC + "." + SUBSCRIPTION,
             EXECUTOR_RUNNING = "executors." + EXECUTOR_NAME + ".running",
-            SCHEDULED_EXECUTOR_OVERRUN = "executors." + EXECUTOR_NAME + ".overrun";
+            SCHEDULED_EXECUTOR_OVERRUN = "executors." + EXECUTOR_NAME + ".overrun",
+            MAXRATE_UPDATES =
+                    "consumers-rate.coordinator.calculation.updates." + GROUP + "." + TOPIC + "." + SUBSCRIPTION,
```

Comment:
i think this should be `consumers-rate.max-rate.*`, also this layout will be hard to extend, how about:

```
coordinator.{group}.{topic}.{sub}.updates
node.{group}.{topic}.{sub}.history.failures
node.{group}.{topic}.{sub}.fetch.failures
```

this way if we want to add another metric we don't need to copy whole metric tree once again